cmd/run.go

```diff
@@ -60,11 +61,27 @@ func Run(cmd *cobra.Command, args []string) {
 			cmd.Help()
 			os.Exit(1)
 		}
-		repos, err := github.GetRepositories(username)
+
+		provider, err := cmd.Flags().GetString("provider")
 		if err != nil {
-			l.Println(err)
+			panic(err)
```

Comment:
Can you make github the default provider instead of panic?

Reply:
It seems that you did it in main.go. If that is the case, you can ignore this comment.